Cargo.lock

```diff
@@ -5737,7 +5738,6 @@ dependencies = [
  "lazy_static",
  "log",
  "lru",
- "matches",
```

Comment:
It's a pity this doesn't actually remove `matches` from our lock file. Looks like it's being kept around because of jsonrpc pulling in an old `url` crate. I'm looking into whether there's anything to be done about that.